buyer-signing-service/src/helpers/buildMethodSigner.js

```diff
@@ -0,0 +1,37 @@
+import { getAddress, getPrivateKey } from './buyer';
+import {
+  createDataBuilder,
+  signTransaction,
+} from '../utils/wibson-lib';
+import config from '../../config';
+
+export default (contract, method) => {
+  return (nonce, gasPrice, params) => {
+    const build = createDataBuilder(contract, method);
```

Comment:
Is there a reason to split the builder from the actual building of params ?

Reply:
The `createDataBuilder` is legacy code. I was reasonable to re-use a dataBuilder among the different signing functions. Now, we have only one signing function and it can have all the functionality needed, but extracting the `createDataBuilder` here is a matter of other PR.